### ISSUE TITLE
Radial Menu on right click?

### ISSUE THREAD

**Issue by tomtachi** [edemaine/cocreate#66]
Can you provide a way to temporarily show up menus right close to your current cursor/pen position?

This is often realized as a radial menu when you right-click or hit a special key. This is helpful as you don't need to move the pens to the very end of the screen each time you switch the tool or change color.

**Comment by edemaine** [comment]
Yes, this is a nice idea: duplicate all the tool/color/width buttons (probably not document-level buttons like grid, save as SVG, etc.) in a popup on appropriate click (I guess right click by default, maybe configurable in a dialog). Should be relatively easy to do, other than figuring out a nice layout...

Meanwhile, note that most tools have keyboard shortcuts (#57 or hover over tool) so that's the fast way to switch tools for now.
